frontend/packages/db-structure/src/deparser/postgresql/operationDeparser.test.ts

```diff
@@ -43,14 +41,14 @@ describe('postgresqlOperationDeparser', () => {
 
       expect(result.errors).toHaveLength(0)
       expect(result.value).toMatchInlineSnapshot(`
-        "CREATE TABLE \"users\" (
-          \"id\" bigint PRIMARY KEY,
-          \"email\" varchar(255) UNIQUE NOT NULL
```

Comment:
Since the test case is intended to convert from column.unique, there is no problem to delete it.
There is a separate test to convert from unique constraints to `ALTER TABLE ~~ ADD CONSTRAINT`.